Update cypher statement transactional requirement in docs (#429)
neo4j/neo4j-documentation: cypher/cypher-docs/src/docs/dev/general/transactions.asciidoc

Any query that updates the graph will run in a transaction.
An updating query will always either fully succeed, or not succeed at all.

> Cypher query should always be executed within transaction.

Reviewer: ```suggestion
The Cypher query should always be executed within a transaction.
```

> Every statement is executed in a transaction context, the query will run inside it, and nothing will be persisted to disk until that transaction is successfully committed.

Reviewer: ```suggestion
Every statement is executed within the context of the transaction, and nothing will be persisted to disk until that transaction is successfully committed.
```

> Following illustrate sequence of operations that can be used to have multiple queries be committed as a single transaction:

Reviewer: ```suggestion
The following illustrates a sequence of operations that can be used to have multiple queries be committed as a single transaction:
```
